specification/ml/start_datafeed/MlStartDatafeedRequest.ts

```diff
@@ -22,7 +22,7 @@ import { Id } from '@_types/common'
 import { Duration, DateTime } from '@_types/Time'
 
 /**
- * Starts one or more datafeeds.
+ * Start one or more datafeeds.
```

Comment:
I think this could be simplified:
```suggestion
 * Start datafeeds.
```